xml/gl.xml

```diff
@@ -12942,15 +12942,15 @@ typedef unsigned int GLhandleARB;
         <command>
             <proto>void <name>glClearBufferData</name></proto>
             <param group="BufferStorageTarget"><ptype>GLenum</ptype> <name>target</name></param>
-            <param group="InternalFormat"><ptype>GLenum</ptype> <name>internalformat</name></param>
+            <param group="SizedInternalFormat"><ptype>GLenum</ptype> <name>internalformat</name></param>
```

Comment:
Parameter group changes are breaking changes. I guess in this instance it could be justified in that all the enumerators that can be passed into this function have been marked with this new group, and that the groups aren't used for Khronos purposes so I suppose this is fine.

I don't know how breaking this will be for other downstream consumers though that generate strongly-typed enums using these attributes - perhaps it could be a good idea to establish a proper policy on these kinds of changes in the future.

Reply:
If passing any InternalFormat before was actually wrong, and then SizedInternalFormat actually does the right thing, then this is technically a break but more importantly this is a bugfix, which generally takes priority over absolutely never doing a break.

Speaking as a person who spent all last week rebuilding their Rust GL bindings generator from scratch: I have strong typing on the GLenum / GLbitmask split, but not on the groups within GLenum (those are all just type aliases). When I asked the other guy who also uses my bindings directly they actually voted for "just be as direct as possible, don't try to fix the typing on anything if you can avoid it".

Reply:
Yeah I've generally taken the same approach as well, but there's no established policy for breaks like this, especially considering it's not breaking for Khronos directly but breaking for a lot of their downstream consumers, including members such as Google and its ANGLE project (cc @null77)